docs/users.rst

```diff
@@ -490,6 +490,17 @@ version
 This key is no longer used by OSBS and is only kept in the schema for backwards
 compatibility.
 
+
+set_release_env
+~~~~~~~~~~~~~~~
+
+Optional string.  If set, osbs-client will modify each stage of the image's 
+Dockerfile, adding an ENV statement at the end of the FROM section.  The
```

Comment:
Maybe instead of "at the end of the FROM section" it should be "immediately following the FROM statement". Then image owners know they are free to use that environment variable at any time.